x-pack/plugins/infra/public/components/waffle/node_context_menu.tsx

```diff
@@ -37,7 +38,9 @@ export const NodeContextMenu = injectI18n(
       ? getNodeDetailUrl({
           nodeType,
           nodeName,
-          from: timeRange.from,
+          from: moment(timeRange.to)
+            .subtract(1, 'hour')
+            .valueOf(),
```

Comment:
I think that silently discarding `timeRange.from` and hardcoding a 1 hour range can lead to confusion later why the `NodeContextMenu` component doesn't generate links honouring the timerange it is passed in.

Would it make sense to do this calculation in `public/components/waffle/node.tsx` instead and pass the correct time range to `NodeContextMenu`, or alternatively, only pass `to` as a parameter?